modules/sorts/test/test.cpp

```diff
@@ -0,0 +1,12 @@
+#define BOOST_TEST_DYN_LINK
+#define BOOST_TEST_MODULE TestSuiteName
```

Comment:
You should give this test suite (and the test below) a useful name to avoid name collisions.